# From laptop camera

image_width: 640
image_height: 480
camera_name: laptop
camera_matrix:
  rows: 3
  cols: 3
  data: [615.373519771983, 0, 331.8001629831776, 0, 614.2095128543981, 245.2891524835611, 0, 0, 1]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.1587533999581577, -0.22857219064868, 0.006387477363770676, 0.003118364787494989, 0]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
projection_matrix:
  rows: 3
  cols: 4
  data: [634.6784057617188, 0, 333.6335487578472, 0, 0, 632.6680297851562, 247.8841900257466, 0, 0, 0, 1, 0]

# Generic example from http://wiki.ros.org/camera_calibration_parsers

# image_width: 2448
# image_height: 2050
# camera_name: prosilica
# camera_matrix:
#   rows: 3
#   cols: 3
#   data: [4827.94, 0, 1223.5, 0, 4835.62, 1024.5, 0, 0, 1]
# distortion_model: plumb_bob
# distortion_coefficients:
#   rows: 1
#   cols: 5
#   data: [-0.41527, 0.31874, -0.00197, 0.00071, 0]
# rectification_matrix:
#   rows: 3
#   cols: 3
#   data: [1, 0, 0, 0, 1, 0, 0, 0, 1]
# projection_matrix:
#   rows: 3
#   cols: 4
#   data: [4827.94, 0, 1223.5, 0, 0, 4835.62, 1024.5, 0, 0, 0, 1, 0]
